Added additional information to Coupon Codes page (#1700)
magento/merchdocs: src/marketing/price-rules-cart-coupon.md

1. When complete, click <span class="btn">Generate</span>.

>    `Message is added to queue, wait to get your coupons soon` message is appeared. Make sure that cron is running on your server.

Reviewer: ```suggestion
   The system displays `Message is added to queue, wait to get your coupons soon`. Make sure that cron is running on your server.
```

>    After the cron job was processed, the list of generated codes appears below.

Reviewer: ```suggestion
   After the cron job was processed, the list of generated codes appears.
```

   ![Cart price rule - generated coupon codes]({% link marketing/assets/coupons-generated.png %}){: .zoom}
   _Generated Codes_

|Field|Description|
|--- |--- |
|Coupon Code|A unique code of coupon that was created and can be used for receiving special conditions.|
|Created|The date when the coupon code was created.|
|Used|Indicates if the coupon was used.|
|Times Used|Indicates how many times the coupon code was used.|

> Coupon codes can be exported to CSV or Excel XML files. For doing that select the needed file format and click <span class="btn">Export</span>.

Reviewer: ```suggestion
You can export coupon codes to a CSV or Excel XML file by selecting the file format and clicking <span class="btn">Export</span>.
```

> To delete coupon codes, select needed codes from the list, select `Delete` from the **Actions** drop-down and click <span class="btn">Submit</span>.

Reviewer: ```suggestion
To delete coupon codes, select one or more codes from the list, select `Delete` from the **Actions**  selector, and click <span class="btn">Submit</span>.
```
